bolt-sidecar/src/driver.rs

```diff
@@ -90,13 +90,17 @@ impl SidecarDriver<StateClient, PrivateKeySigner> {
 
         // Constraints are signed with a BLS private key
         let constraint_signer = SignerBLS::Local(LocalSigner::new(
-            opts.signing.private_key.clone().expect("local signer").0,
+            opts.constraint_signing
+                .constraint_private_key
+                .clone()
+                .expect("local constraint signing key")
+                .0,
             opts.chain,
         ));
 
         // Commitment responses are signed with a regular Ethereum wallet private key.
         // This is now generated randomly because slashing is not yet implemented.
-        let commitment_signer = PrivateKeySigner::random();
+        let commitment_signer = PrivateKeySigner::from_slice("".as_bytes().to_vec().as_slice())?;
```

Comment:
Why are we initialising the `PrivateKeySigner` from an empty string?

Reply:
sorry was a placeholder